.gitignore

```diff
@@ -8,7 +8,6 @@ __pycache__/
 *$py.class
 
 # folders
-data/
```

Comment:
@m090009, I'm not sure why you had this ignored originally.  Everything fails to run without it and the files aren't very big.  If you have another reason, feel free to ignore this particular commit.